tests/components/neato/test_init.py

```diff
@@ -85,10 +89,12 @@ def mock_controller_login():
 
 async def test_config_entries_not_in_sync(hass, config_flow, hub):
     """The config entry and configuration.yaml are not in sync."""
-    MockConfigEntry(domain=NEATO_DOMAIN, data=DIFFERENT_CONFIG).add_to_hass(hass)
+    MockConfigEntry(domain=NEATO_DOMAIN,
+                    data=DIFFERENT_CONFIG).add_to_hass(hass)
 
     assert hass.config_entries.async_entries(NEATO_DOMAIN)
-    assert await async_setup_component(hass, NEATO_DOMAIN, {NEATO_DOMAIN: VALID_CONFIG})
+    assert await async_setup_component(hass, NEATO_DOMAIN,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)